brightlayer-ui/angular-design-patterns
# Add new patterns to README.md

Fixes # .


#### Changes proposed in this Pull Request:
Update README.md to include newly added patterns
- Action List With Inline Actions
- Action List In Button Panel
- Action List In Panel Header
- Action List With Local Actions
- List As A Tree Structure


#### Screenshots:


#### To Test:
- 
Check all the newly added descriptions and links here: https://github.com/brightlayer-ui/angular-design-patterns/tree/feature/update-readme-examples-list
